k4MarlinWrapper/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ gaudi_add_module(k4MarlinWrapperPlugins
     src/components/LcioEventOutput.cpp
   LINK
     Gaudi::GaudiAlgLib
-    ${Marlin_LIBRARIES}
+    Marlin
```

Comment:
This just makes cmake add `-lMarlin`, so you would have to add `-L /path/to/marlin/lib` as well.

Or you make Marlin create targets.

Or you add `-Wl,--as-needed`
But make sure that doesn't drop libraries that we need for the EDM at runtime, but not at linktime?

Reply:
Yeah forget about this PR for now, I should just close it and have to find how to solve my issues in a different way I think